script/predeployment/DeployRelayers.s.sol

```diff
@@ -106,3 +106,47 @@ contract DeployWstethEthChainlinkRelayerMainnet is CommonMainnet {
     vm.stopBroadcast();
   }
 }
+
+// BROADCAST
+// source .env && forge script DeployRethPtToSyPendleRelayerMainnet --with-gas-price 2000000000 -vvvvv --rpc-url $ARB_MAINNET_RPC --broadcast --verify --etherscan-api-key $ARB_ETHERSCAN_API_KEY
```

Comment:
- Update to use cast wallet commands, as demonstrated in https://github.com/open-dollar/od-relayer/blob/0bb0d95a6c7b88d6f76092a3abe3722149982225/script/DeployOracle.s.sol